cogs5e/models/character.py

```diff
@@ -209,6 +209,12 @@ async def from_bot_and_ids(cls, bot, owner_id: str, character_id: str):
         cls._cache[owner_id, character_id] = inst
         return inst
 
+    @classmethod
+    async def serialize_character_from_dict(cls, owner_id, character_dictionary):
```

Comment:
`deserialize` might make more sense here than `serialize`, since the result is an object instance rather than a string or dictionary.

Reply:
Ah, good point